packages/flutter/lib/src/material/checkbox_list_tile.dart

```diff
@@ -57,6 +57,78 @@ import 'theme_data.dart';
 /// ```
 /// {@end-tool}
 ///
+/// ## CheckboxListTile isn't exactly what I want
+///
+/// If the way ListTile pads and positions its elements isn't quite what you're
+/// looking for, you can create custom labeled radio widgets by combining it
```

Comment:
"create custom labeled radio widgets" aren't we talking about check boxes here?

combining it => combining [Checkbox]